ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/EeaSendTransaction.java

```diff
@@ -128,16 +127,21 @@ private PrivateTransactionBuilder defaultTransaction() {
         .withGasPrice("0x9184e72a000")
         .withGas("0x76c0")
         .withTo("0xd46e8dd67c5d32be8058bb8eb970870f07244567")
-        .withValue(VALUE)
-        .withData(DATA)
-        .withPrivateFrom(PRIVATE_FROM)
-        .withPrivateFor(PRIVATE_FOR)
-        .withRestriction(RESTRICTED);
+        .withValue(DEFAULT_VALUE)
+        .withData(
+            "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675")
+        .withPrivateFrom("ZlapEsl9qDLPy/e88+/6yvCUEVIvH83y0N4A6wHuKXI=")
```

Comment:
just curious, which does above default private from resolves to?

Reply:
It was just public key that got generated when I originally tested the this long ago, so the node it corresponds to doesn't exist anymore.